includes/install.php

```diff
@@ -151,7 +151,8 @@ function give_run_install() {
 			'v230_move_donor_note',
 			'v230_move_donation_note',
 			'v230_delete_donor_wall_related_donor_data',
-			'v230_delete_donor_wall_related_comment_data'
+			'v230_delete_donor_wall_related_comment_data',
+			'v240_update_form_goal_progress'
```

Comment:
```suggestion
			'v240_update_form_goal_progress',
```
End each item in array with a comma according to WPCS.

Reply:
@kevinwhoffman Done.